package.json

```diff
@@ -51,8 +51,8 @@
     "@google-cloud/precise-date": "^3.0.0",
     "@google-cloud/projectify": "^3.0.0",
     "@google-cloud/promisify": "^2.0.0",
-    "@opentelemetry/api": "^1.0.0",
-    "@opentelemetry/semantic-conventions": "~1.3.0",
+    "@opentelemetry/api": "~1.1.0",
```

Comment:
any particular reason these are locked down from `^`?

Reply:
Yeah, it had to do with Node 12 compatibility. I found that there were some semver issues, in practice, though I'll be revisiting this for the 4.x version of the PR, so I'll probably recheck these for 3.x too.